lib/platform/bitbucket/utils.ts

```diff
@@ -56,6 +57,36 @@ export function repoInfoTransformer(repoInfoBody: RepoInfoBody): RepoInfo {
   };
 }
 
+export type BitbucketMergeStrategy = 'fast_forward' | 'merge_commit' | 'squash';
```

Comment:
Please move new  type and interface definitions to `./types.ts` and do a `import type`

Reply:
`lib/platform/bitbucket/types.ts` does not yet exist. Would you prefer me to move *all* type and interface definitions in this file into that new file, or just the ones added here?

Reply:
@viceice, could you elaborate on if you'd prefer:

* Create this new file, add only my stuff in it (IE: Keep the PR small)
* Create this new file, add all the appropriate stuff into it (IE: Help the code conform to style)

Reply:
for this pr only move new types and interfaces. Also prefer interfaces if possible.

Reply:
Resolved in a4e10f9